### ISSUE TITLE
Ghost Pokemon at Pokemon Tower with Shadow Tag (before Silph Scope)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [todtim2/universal-pokemon-randomizer#3]
```
What steps will reproduce the problem?
1. Do not have Silph Scope
2. Go to Pokemon Tower at Lavender Town
3. Fight a Pokemon with Shadow Tag

What is the expected output? What do you see instead?
Cannot run, cannot do anything. Only thing is to reset game.

What version of the product are you using? On what operating system?
Pokemon LeafGreen. I expect this problem to also be in FireRed, Red, Blue, 
Yellow.

Please provide any additional information below.

Maybe you can disable Shadow Tag for Pokemon in the Pokemon Tower only before 
you get the Silph Scope.

```

Original issue reported on code.google.com by `jklph...@gmail.com` on 24 Jan 2013 at 10:32

**Comment by GoogleCodeExporter** [comment]
```
Shadow Tag, as it is an ability, does not exist in Red/Blue/Yellow so this 
problem is only only with FireRed and LeafGreen.

This is an interesting conundrum, one which I'll try to patch in the manner you 
suggested for 1.1.2, coming out soon.
```

Original comment by `dabomstew` on 24 Jan 2013 at 12:56
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Should be resolved as of 1.1.2. Naturally, you'll have to randomize a new ROM 
with 1.1.2 to have the issue completely solved if you haven't passed Pokemon 
Tower yet!
```

Original comment by `dabomstew` on 24 Jan 2013 at 2:11
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Oh how silly of me, I forgot R/B/Y did not have abilities.

And thank you for your hard work! I love the Randomizer.
```

Original comment by `jklph...@gmail.com` on 25 Jan 2013 at 3:11
